src/utils/getPriceGuidance.ts

```diff
@@ -1,10 +1,13 @@
 import { reduce } from "lodash"
-const { metaphysics } = require("../lib/metaphysics")
+import metaphysics from "../lib/metaphysics"
+// I notice when I don't require the metaphysics module
+// like below the test git satfails
+// const metaphysics = require("../lib/metaphysics").metaphysics
```

Comment:
@eessex do you know why the tests require the module to be loaded in using the `require` method?

Reply:
Hm I haven't experienced this issue, just checked this pr and it looks like we were able to use the import like above-- https://github.com/artsy/kaws/pull/78/files